Mark9Mbugua/SendIT23
# #161846017 Test endpoints

**What does this PR do?**
It creates tests for the endpoints

**Description of Task to be completed?**
Create a tests folder in the root directory and create a test_parcel.py file for writing tests to check if they are working.Write tests first without writing the actual code to make sure that the tests fail. Write the code and run pytest in the terminal. The tests should now pass.

** How should this be manually tested?**
Clone this repo in a local repository. Navigate to the test_parcel.py file in the tests directory. Ensure that pytest is installed. Run pytest in the terminal.

** Any background context you want to provide?**
Good tests should pass.

**What are the relevant pivotal tracker stories?**
#161846017
